assets/src/setupCustomSidebar.js

```diff
@@ -1,45 +1,48 @@
-import { registerPlugin } from "@wordpress/plugins";
-import { CampaignSidebar } from './components/Sidebar/CampaignSidebar';
+import { registerPlugin } from '@wordpress/plugins';
+import { PageHeaderSidebar } from './components/Sidebar/PageHeaderSidebar';
+import { CampaignThemeSidebar } from './components/Sidebar/CampaignThemeSidebar';
 import { ActionSidebar } from './components/Sidebar/ActionSidebar';
 
-const sidebarForPostType = ( postType ) => {
-  switch ( postType ) {
-    case 'campaign':
-      return {
-        id: CampaignSidebar.getId(),
-        icon: CampaignSidebar.getIcon(),
-        render: CampaignSidebar,
-      };
-    case 'p4_action':
-      return {
-        id: 'planet4-action-sidebar',
-        render: ActionSidebar,
-      };
-    default:
-      return null;
+const sidebarsForPostType = postType => {
+  switch (postType) {
+  case 'campaign':
+    return [
+      PageHeaderSidebar,
+      CampaignThemeSidebar,
```

Comment:
Could you please confirm, both panels are visible on campaign page, or might be my local issue 🤔